waspc/cli/src/Wasp/Cli/Command/Build.hs

```diff
@@ -41,16 +49,22 @@ build = do
     cliSendMessageC $ Msg.Success "Successfully cleared the contents of the .wasp/build directory."
 
   cliSendMessageC $ Msg.Start "Building wasp project..."
-  buildResult <- liftIO $ buildIO waspProjectDir buildDir
-  case buildResult of
-    Left compileError -> throwError $ CommandError "Build failed" compileError
-    Right () -> cliSendMessageC $ Msg.Success "Code has been successfully built! Check it out in .wasp/build directory."
-  cliSendMessageC $ Msg.Warning "Build warning" alphaWarningMessage
+  (warnings, errors) <- liftIO $ buildIO waspProjectDir buildDir
+
+  liftIO $ printCompilationResult (warnings, errors)
+  if null errors
+    then do
+      cliSendMessageC $
+        Msg.Success "Wasp project has been successfully built! Check it out in .wasp/build directory."
+      cliSendMessageC $ Msg.Warning "Wasp is in alpha" alphaWarningMessage
```

Comment:
Because we use both `printCompilationResult` above and explicitly call `cliSendMessageC` here with a warning, it can happen that warnings are not grouped. Is that ok? I think it probably is, but just wanted to note:
<img width="1225" alt="Screenshot 2022-10-25 132504" src="https://user-images.githubusercontent.com/523636/197841419-9d0acac2-f862-4726-a478-6471b04bb3f8.png">

Reply:
Actually, I just realized on another branch that this is existing behavior. Please disregard :)

Reply:
Yup, this is how we did it -> this alpha warning is a separate, standalone thing. So I would worry about it too much for now, as we will also remove it relatively soon.
Actual compiler warnings are grouped together, the ones that come from analyzer / generator. This is a special case.